datafusion/sql/tests/sql_integration.rs

```diff
@@ -397,7 +397,7 @@ CopyTo: format=csv output_url=output.csv options: ()
 
 #[test]
 fn plan_explain_copy_to() {
-    let sql = "EXPLAIN COPY test_decimal to 'output.csv'";
+    let sql = "EXPLAIN COPY test_decimal to 'output.csv' STORED AS CSV";
```

Comment:
I think we should continue to support the original syntax here, which is in my opinion much more clear and concise.

Reply:
I agree with this, I can made STORED AS optional.